bats/1pksupportedtypes.bats

```diff
@@ -3,60 +3,52 @@ load $BATS_TEST_DIRNAME/helper/common.bash
 
 setup() {
     setup_common
-    dolt table create -s=`batshelper 1pksupportedtypes.schema` test
+    dolt sql < $BATS_TEST_DIRNAME/helper/1pksupportedtypes.schema
 }
 
 teardown() {
     teardown_common
 }
 
 @test "dolt table put-row with all types then examine table" {
-    run dolt table put-row test pk:0 int:1 string:foo boolean:true float:1.11111111111111 uint:346 uuid:123e4567-e89b-12d3-a456-426655440000
-    [ "$status" -eq 0 ] 
-    [ "$output" = "Successfully put row." ]
-    run dolt table select test
+    run dolt sql -q "insert into test values (0, 1, 'foo', true, 1.11111111111111, 346, '123e4567-e89b-12d3-a456-426655440000')"
+    [ "$status" -eq 0 ]
+    run dolt sql -q "select * from test"
     [ "$status" -eq 0 ]
     [ "${#lines[@]}" -eq 5 ]
 }
 
 @test "boolean 1,0,true,false inserts and examine table" {
-    run dolt table put-row test pk:0 int:1 string:foo boolean:1 float:1.11111111111111 uint:346 uuid:123e4567-e89b-12d3-a456-426655440000
+    run dolt sql -q "insert into test values (0, 1, 'foo', 1, 1.11111111111111, 346, '123e4567-e89b-12d3-a456-426655440000')"
     [ "$status" -eq 0 ]
-    [ "$output" = "Successfully put row." ]
-    run dolt table select test boolean
+    run dolt sql -q "select * from test" boolean
     [ "$status" -eq 0 ]
     [[ "${lines[1]}" =~ "boolean" ]] || false
-    # Can't get the [ "${lines[1]}" = "true" ] to return true. Going regex instead
-    [[ "${lines[3]}" =~ "true" ]] || false
-    run dolt table put-row test pk:0 int:1 string:foo boolean:0 float:1.11111111111111 uint:346 uuid:123e4567-e89b-12d3-a456-426655440000
+    [[ "${lines[3]}" =~ "1" ]] || false
```

Comment:
I think this can now be turned to an equality not a regex. Try:
[ "${lines[3]}" -eq 1 ]

Reply:
I talked myself out of this comment. Ignore.